packages/aws-cdk-lib/aws-opensearchservice/lib/domain.ts

```diff
@@ -1556,6 +1566,7 @@ export class Domain extends DomainBase implements IDomain, ec2.IConnectable {
           : undefined,
         instanceCount,
         instanceType,
+        multiAzWithStandbyEnabled,
```

Comment:
```suggestion
        multiAzWithStandbyEnabled: props.capacity?.multiAzWithStandbyEnabled,
```

I don't think there is any reason to explicitly set this to false.

Reply:
I've added a feature flag that defaults `multiAzWithStandbyEnabled` to `true`).
This is causing some integration tests to fail.
Let me know if the implementation is correct so I can proceed to fix the failing tests.